src/code/z_debug.c

```diff
@@ -79,7 +79,7 @@ void func_8006376C(u8 x, u8 y, u8 colorId, const char* text) {
 }
 
 // Draw Text
-void func_80063828(GfxPrint* gfxPrint) {
+void func_80063828(GfxPrint* printer) {
```

Comment:
You might think it's out of scope for this PR, but it looks like there's a few functions ready to rename in this file.

Reply:
Yeah looks like you're right, a lot of things could be named or improved in this file, but it probably needs its own documentation pass to really name things right so I'd prefer to leave it out of this PR